CHANGELOG.md

```diff
@@ -8,10 +8,11 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [0.40.0](https://github.com/usgs/waterdataui/compare/waterdataui-0.39.0...waterdataui-0.40.0) - 2021-01-06
 ### Added
-- Instantaneous Values data availability information page.
-- Additional analytics tracking.
-- Added an inventory data menu on the monitoring location page which is context specific for the site.
+- Instantaneous Values data availability information page
+- Additional analytics tracking
+- Added an inventory data menu on the monitoring location page which is context specific for the site
 - Added social media icons in the footer
+- Added a selection for Celsius converted to Fahrenheit to the parameter table
```

Comment:
You should add this under [Unreleased] header

Reply:
UGH. duh. Fixed.